README.md

```diff
@@ -65,6 +65,12 @@ Enables secondary MongoDB database configuration.
 ```
 
 Enable and installs pritunl plugins
+```yaml
+    pritunl_reverse_proxy: True
+    pritunl_server_port: 3000
```

Comment:
@raphapr it’s very import to define default values for roles variables in **defaults/main.yml** .
I did not see these reverse proxies vars there.

Reply:
You're right!